dialog.js

```diff
@@ -1,39 +1,39 @@
 var dialogState = {
-    isRender: false,
```

Comment:
Be consistent about indentation. Is this intended?

Reply:
this is after alt+shift+f   ...

Reply:
Then make the whole code is indented with 4 spaces.